packages/cms/src/components/variables/VarTable.jsx

```diff
@@ -16,9 +17,15 @@ export default class VarTable extends Component {
           <tbody className="cms-var-table-body">
             {Object.keys(dataset).map(k =>
               <tr className="cms-var-table-row" key={ k }>
-                <td className="cms-var-table-cell">
-                  { k }:
-                </td>
+                {dupes.includes(k) 
+                  ? <td style={{color: "red"}} className="cms-var-table-cell">
+                    { k }:
+                  </td>
```

Comment:
@perpetualgrimace need your review with this highlighting of a dupe variable

Reply:
@jhmullen

`<td className="cms-var-table-cell warning">`

`.cms-var-table-cell.warning { @mixin error-text; }`

Reply:
Fixed